.vscode/settings.json

```diff
@@ -5,5 +5,6 @@
 	"editor.defaultFormatter": "esbenp.prettier-vscode",
 	"editor.formatOnSave": true,
 	"eslint.rules.customizations": [{ "rule": "*", "severity": "warn" }],
+	"eslint.probe": ["javascript", "javascriptreact", "typescript", "typescriptreact", "html", "vue", "markdown", "jsonc", "json"],
```

Comment:
Nitpick: could you please alphabetize this list? I know it's very nitpicky but I find it much easier to scan through lists when they're in order. 😄

Reply:
Okay, will be making these changes in sometime